types/lib/client-options.d.ts

```diff
@@ -192,6 +192,15 @@ export interface IClientSubscribeOptions {
     userProperties?: UserProperties
   }
 }
+export interface IClientSubscribeProperties {
+  /*
+  *  MQTT 5.0 properies object of subscribe
+  * */
+  properties?: {
```

Comment:
Are we actually handling this somewhere in code?

Reply:
What does "in code" mean? If it means MQTT.js internal code, the answer is no.
However, this is a part of public APIs. MQTT.js users that access MQTT.js APIs via TypeScript require it.
For example, my project using MQTT.js via TypeScript requires setting SubscriptionIdentifier property when subscrine() API calling. Lack of this TypeScript definition, I need to use type unsafe JavaScript code directly.
This definition improve type safety for TypeScript users.